src/Bicep.LangServer/Handlers/BicepCompletionHandler.cs

```diff
@@ -61,7 +61,7 @@ public override Task<CompletionItem> Handle(CompletionItem request, Cancellation
             DocumentSelector = DocumentSelectorFactory.Create(),
             AllCommitCharacters = new Container<string>(),
             ResolveProvider = false,
-            TriggerCharacters = new Container<string>(":", " ", ".", "/", "'", "@")
+            TriggerCharacters = new Container<string>(":", " ", ".", "/", "'", "@", "{")
```

Comment:
This is not related to this change, but I noticed that we provide completions inside single-line empty objects are not resource/module bodies:
![image](https://user-images.githubusercontent.com/16367959/117655459-b0578280-b1c9-11eb-80cb-31a89525cd53.png)

I feel like we should block it and I created https://github.com/Azure/bicep/issues/2597 to track it.